sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from users.factories import UserFactory
+
+CustomUser = get_user_model()
+
+admin_user_type = CustomUser.UserType.ADMIN.name
+employee_user_type = CustomUser.UserType.EMPLOYEE.name
+manager_user_type = CustomUser.UserType.MANAGER.name
+superuser_user_type = "SUPERUSER"
```

Comment:
You don't have to declare new variables just to assign them few lines later. Please delete them from here

Reply:
Everywhere where you used these variables please substitute them with `UserType`s from `CustomUser` model

Reply:
Done. I left _superuser_user_type_,  because there ain't this UserType in CustomUser model.